dapp/src/components/ConnectWalletModal/index.tsx

```diff
@@ -1,37 +1,61 @@
-import React, { useState } from "react"
+import React, { useEffect, useState } from "react"
 import { ModalBody, ModalHeader, ModalCloseButton } from "@chakra-ui/react"
-import { useConnectors, useWalletConnectionError } from "#/hooks"
+import {
+  useConnectors,
+  useIsEmbed,
+  useIsSignedMessage,
+  useWallet,
+  useWalletConnectionError,
+} from "#/hooks"
 import { OrangeKitConnector, BaseModalProps, OnSuccessCallback } from "#/types"
-import { wallets } from "#/constants"
 import withBaseModal from "../ModalRoot/withBaseModal"
 import ConnectWalletButton from "./ConnectWalletButton"
 import ConnectWalletErrorAlert from "./ConnectWalletErrorAlert"
 
 export function ConnectWalletModalBase({
   onSuccess,
   withCloseButton = true,
+  isReconnecting,
 }: {
   onSuccess?: OnSuccessCallback
+  isReconnecting?: boolean
 } & BaseModalProps) {
+  const { isEmbed } = useIsEmbed()
+  const { onDisconnect } = useWallet()
   const connectors = useConnectors()
   const enabledConnectors = connectors.map((connector) => ({
     ...connector,
-    isDisabled: !wallets.SUPPORTED_WALLET_IDS.includes(connector.id),
+    isDisabled: false,
```

Comment:
@kkosiorowska do we still want to be able to enable/disable wallets? I also wanted to point out that we can't control the `Unisat` wallet via environment variables.

Reply:
>@kkosiorowska do we still want to be able to enable/disable wallets? I 

I think it is no longer necessary. It would also be good to get rid of feature flags that are no longer needed.